lib/pubsub/index.js

```diff
@@ -163,6 +163,123 @@ PubSub.prototype.createTopic = function(name, callback) {
 };
 
 /**
+ * Create a subscription to this topic. You may optionally provide an object to
+ * customize the subscription.
+ *
+ * Your provided callback will either be invoked with an error object, if an API
+ * error occurred, or a {@linkcode module:pubsub/subscription} object.
+ *
+ * @throws {Error} If a Topic instance or topic name is not provided.
+ * @throws {Error} If a subName is not provided.
+ *
+ * @param {module:pubsub/topic|string} - topic - The Topic to create a
+ *     subscription to.
+ * @param {string} subName - The name of the subscription.
+ * @param {object=} options - Configuration object.
+ * @param {number} options.ackDeadlineSeconds - The maximum time after receiving
+ *     a message that you must ack a message before it is redelivered.
+ * @param {boolean} options.autoAck - Automatically acknowledge the message once
+ *     it's pulled. (default: false)
+ * @param {number} options.interval - Interval in milliseconds to check for new
+ *     messages. (default: 10)
+ * @param {boolean} options.reuseExisting - If the subscription already exists,
+ *     reuse it. The options of the existing subscription are not changed. If
+ *     false, attempting to create a subscription that already exists will fail.
+ *     (default: false)
+ * @param {function} callback - The callback function.
+ *
+ * @example
+ * // Without specifying any options.
+ * var topic = pubsub.topic('messageCenter');
+ * var name = 'newMessages';
+ *
+ * pubsub.subscribe(topic, name, function(err, subscription, apiResponse) {});
+ *
+ * // With options.
+ * pubsub.subscribe(topic, name, {
+ *   ackDeadlineSeconds: 90,
+ *   autoAck: true,
+ *   interval: 30
+ * }, function(err, subscription, apiResponse) {});
+ */
+PubSub.prototype.subscribe = function(topic, subName, options, callback) {
+  if (!util.is(topic, 'string') && !(topic instanceof Topic)) {
+    throw new Error('A Topic is required for a new subscription.');
+  }
+
+  if (!subName) {
+    throw new Error('A subscription name is required for a new subscription.');
+  }
+
+  if (!callback) {
+    callback = options;
+    options = {};
+  }
+
+  options = options || {};
+
+  if (util.is(topic, 'string')) {
+    topic = this.topic(topic);
+  }
+
+  var body = {
+    topic: topic.name
+  };
+
+  if (options.ackDeadlineSeconds) {
+    body.ackDeadlineSeconds = options.ackDeadlineSeconds;
+  }
+
+  var subscription = topic.subscription(subName, options);
```

Comment:
> `topic.subscription(...`

`this.subscription(...` will accomplish the same thing and looks less wrong. But I might be nitpicking this.